editor.html

```diff
@@ -227,9 +227,8 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                 </div>
             </div>
             <div class="vbox" id="script-box">
-                <div id="script-name" class="flex1" contenteditable
-                    tabindex="10">
-                </div>
+                <label for="script-name">Script Name</label>
```

Comment:
The size of the text in the input box changes at lower resolutions. Could we reduce the text of this label as well?